tests/Aspire.MySqlConnector.Tests/ConformanceTests.cs

```diff
@@ -12,12 +13,12 @@
 
 namespace Aspire.MySqlConnector.Tests;
 
-public class ConformanceTests : ConformanceTests<MySqlDataSource, MySqlConnectorSettings>
+public class ConformanceTests : ConformanceTests<MySqlDataSource, MySqlConnectorSettings>, IClassFixture<MySqlContainerFixture>
 {
-    private const string ConnectionSting = "Host=localhost;Database=test_aspire_mysql;Username=root;Password=password";
-
-    private static readonly Lazy<bool> s_canConnectToServer = new(GetCanConnect);
-
+    private readonly MySqlContainerFixture _containerFixture;
+    private string ConnectionSting => RequiresDockerTheoryAttribute.IsSupported
+                                        ? _containerFixture.GetConnectionString()
+                                        : "Host=localhost;Database=test_aspire_mysql;Username=root;Password=password";
```

Comment:
```suggestion
                                        : "Server=localhost;User ID=root;Password=password;Database=test_aspire_mysql";
```

Don't know if it matters, but the "normalized" version of a MySqlConnector connection string would use different keywords (`Server` not `Host`) and a slightly different order.

Reply:
Indeed, I was hitting that - https://github.com/dotnet/aspire/commit/491a14b473964e06ef458273d7fe43d5bd43df88